packages/opentelemetry-api/src/trace/tracer.ts

```diff
@@ -39,9 +39,10 @@ export interface Tracer {
    * Starts a new {@link Span}.
    * @param name The name of the span
    * @param [options] SpanOptions used for span creation
+   * @param [context] Context to use to extract parent
    * @returns Span The newly created span
    */
-  startSpan(name: string, options?: SpanOptions): Span;
+  startSpan(name: string, options?: SpanOptions, context?: Context): Span;
```

Comment:
Is it possible to add `Context` as a part of `SpanOptions`,  in order to avoid `propagation.inject(metadata, **undefined**, (c, k, v) => c.set());` in grpc plugin?

Reply:
the grpc plugin was a typo and that function is no longer there

Reply:
That is part of the propagators api and is unrelated to the startspan signature